sdk/messaging/azeventhubs/consumer_client.go

```diff
@@ -220,34 +217,66 @@ func (cc *ConsumerClient) Close(ctx context.Context) error {
 	return cc.namespace.Close(ctx, true)
 }
 
-func getOffsetExpression(startPosition StartPosition) string {
+func getOffsetExpression(startPosition StartPosition) (string, error) {
 	lt := ">"
 
 	if startPosition.Inclusive {
 		lt = ">="
 	}
 
+	var errMultipleFieldsSet = errors.New("Only a single start point can be set: Earliest, EnqueuedTime, Latest, Offset, or SequenceNumber")
```

Comment:
Don't capitalize the first letter in error messages (the linter will bark at this).

Reply:
Gah, and I _know_ this and still did it.